go/store/nbs/file_table_persister.go

```diff
@@ -65,6 +60,10 @@ type fsTablePersister struct {
 	// remove the entry from this map when we are done processing the temp file
 	// or else this map will grow without bound.
 	curTmps map[string]struct{}
+	dir     string
+	// Protects the following two maps.
```

Comment:
This comment doesn't make sense anymore after changing the position. Suggesting instead: 

```suggestion
	// Protects the toKeep and curTmps maps.
```